inference-engine/src/inference_engine/include/openvino/runtime/gpu/ocl/va.hpp

```diff
@@ -7,14 +7,14 @@
  * shared Video Acceleration device contexts
  * and shared memory tensors which contain Video Acceleration surfaces
  *
- * @file openvino/runtime/gpu/va.hpp
+ * @file openvino/runtime/gpu/ocl/va.hpp
  */
 #pragma once
 
```

Comment:
maybe we also need to add guard that is not supported on Windows? Or it works on all platforms?

Reply:
AFAIK, it works on linux/BSD based systems only. In the plugin we use `#ifndef _WIN32` condition for VAAPI path, so I added it to the header as well to be aligned with impl.